sdcm/cluster.py

```diff
@@ -2852,6 +2852,10 @@ def run_in_thread():
             while current - start < duration:
                 loader_node = random.choice(self.nodes)
                 db_node = random.choice(db_nodes)
+                if db_node.remoter and not db_node.remoter.is_up():
+                    self.log.warning("Chosen node is down. Skip Request")
+                    time.sleep(interval)
+                    continue
                 self.log.info('Fullscan start on node {} from loader {}'.format(db_node.name, loader_node.name))
```

Comment:
Can we add above new code after termination_event checking? it might delay the look to break when termination_event is already set.

For example, In cleanup stage, nodes are destroyed, and termination_event is set, but the new code will bright a redundant sleep.

Reply:
good catch Amos :+1:

Reply:
Thanks. Fixed